resources/ViewAuthority.html

```diff
@@ -1,10 +1,30 @@
 <!-- Landing page for viewing authority -->
 <!-- See src/authority.ts:129 -->
 
-<a href="/getauthority?email=me"><button>Get your own authority level</button></a>
-<hr />
-<h4>Get authority level for an email</h4>
-<form action="/getauthority">
-    <label for="email">Email address:</label> <input type="text" id="email" name="email"><br />
-    <input type="submit" value="Submit">
-</form>
+<!DOCTYPE html>
+
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Scouting App | Get Authority</title>
+    <link rel="stylesheet" href="/css/ViewAuthority.css">
+  </head>
+
+  <body>
+    <div class="content-container">
+      <h4 id="viewauthority-title">Get authority level for an email</h4>
+      <form action="/getauthority">
+          <div class="label-input-div">
+            <label for="email">Email address:</label> <input type="text" id="email" name="email"><br />
+          </div>
+          <input class="emailauthority-submit submit" type="submit" value="Submit" placeholder="Email">
+      </form>
+      <a href="/getauthority?email=me"><button class="getauthority-submit submit">Get your own authority level</button></a>
+    </div>
+
+    <script> </script>
```

Comment:
What is this for?

Reply:
We shouldn't need `<script>` tags at all; I don't think this site does anything that needs client-side JavaScript.

Reply:
Oh yeah. So google chrome has this weird glitch with CSS animation where animations run on page load. There are many different ways to prevent this glitch, but the easiest way to do it is to put empty script tags at the bottom of the html document.

Reply:
I don't see any animations at all in Chrome with the `<script>` tag.

Reply:
That being said, this is an actual bug, so I guess I'm fine with the tags.

However, you should probably add a comment explaining it and linking to https://bugs.chromium.org/p/chromium/issues/detail?id=332189#c20.

Reply:
k

Reply:
(You haven't actually done this yet.)

Reply:
Sorry about that, lemme do this rn

Reply:
Alright, just did it. Lemme know if it shows up for you. New to this whole github thing lol